client/src/components/AddColor.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+const AddColor =()=>{
+    <form onSubmit={saveEdit}>
+          <legend>edit color</legend>
+          <label>
+            color name:
```

Comment:
It looks like this function is missing a return statement. It should look something like
```
const AddColor =()=> {
    return (
        <form onSubmit={saveEdit}>
                <legend>edit color</legend>
                {/* other code here */}
```